linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_tcp_invalid_ratelimit/rule.yml

```diff
@@ -0,0 +1,58 @@
+documentation_complete: true
+
+prodtype: wrlinux1019,rhel7,rhel8
```

Comment:
```suggestion
prodtype: wrlinux1019,rhel7,rhel8,ocp4
```

Reply:
Would this also be applicable to CoreOS? Meaning, is the ``ocp4`` prodtype needed?